SU-SWS/saa_victoria_subtheme
# ADAPT-639: First pass at adding link underline back in for regular li…

…nks but not cta links on default state.

# READY


# Summary
- Add underline back for default state of ONLY regular links. CTA links should not have the underline in their default state.

# Review By (Date)
- 8/27

# Urgency
- medium, accessibility

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Test regular links and cta links, check their default state (regular link with underline, no underline for cta). 
4. On hover, they should all get the underline.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-639
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
